### PR TITLE
Translation of tutorial/part-zero.md 

### PR DESCRIPTION
**Pull request by alexislepresle**
Translation of tutorial/part-zero.md to french

### CODE REVIEW

File: docs/tutorial/part-zero/index.md [hunk 1]
```diff
@@ -1,194 +1,195 @@
...
 
-npm has three distinct components: the npm website, the npm registry, and the npm command line interface (CLI).
+npm a trois composants distincts: le site Web npm, le registre npm et l'interface de ligne de commande (CLI) npm.
 
-- On the npm website, you can browse what JavaScript packages are available in the npm registry.
-- The npm registry is a large database of information about JavaScript packages available on npm.
-- Once you’ve identified a package you want, you can use the npm CLI to install it in your project or globally (like other CLI tools). The npm CLI is what talks to the registry — you generally only interact with the npm website or the npm CLI.
+- Sur le site Web de npm, vous pouvez parcourir les packages JavaScript disponibles dans le registre npm.
+- Le registre npm est une grande base de données d'informations sur les packages JavaScript disponibles sur npm.
+- Une fois que vous avez identifié le package souhaité, vous pouvez utiliser la CLI npm pour l'installer dans votre projet ou globalement (comme les autres outils CLI). La CLI npm est ce qui parle au registre
```
Comment (dantereve) [inline docs/tutorial/part-zero/index.md]:
```suggestion
- Une fois que vous avez identifié le package souhaité, vous pouvez utiliser la CLI npm pour l'installer dans votre projet ou globalement (comme les autres outils CLI). La CLI npm est ce qui parle au registre.
```

File: docs/tutorial/part-zero/index.md [hunk 2]
```diff
@@ -1,194 +1,195 @@
...
-- Learned about the command line and how to use it
-- Installed and learned about Node.js and the npm CLI tool, the version control system Git, and the Gatsby CLI tool
-- Generated a new Gatsby site using the Gatsby CLI tool
-- Ran the Gatsby development server and visited your site locally
-- Downloaded a code editor
-- Installed a code formatter called Prettier
+- Connaître la ligne de commande et comment l'utiliser
+- Installation et apprentissage de Node.js et de l'outil CLI npm, du système de contrôle de version Git et de l'outil CLI Gatsby
+- Généré un nouveau site Gatsby à l'aide de l'outil CLI Gatsby
+- Exécutez le serveur de développement Gatsby et visité votre site localement
```
Comment (dantereve) [inline docs/tutorial/part-zero/index.md]:
```suggestion
- Exécutez le serveur de développement Gatsby et visitez votre site localement
```

File: docs/tutorial/part-zero/index.md [hunk 3]
```diff
@@ -1,194 +1,195 @@
 ---
-title: Set Up Your Development Environment
+title: Configurer votre environnement de développement
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-Before you start building your first Gatsby site, you’ll need to familiarize yourself with some core web technologies and make sure that you have installed all required software tools.
+Avant de commencer à créer votre premier site Gatsby, vous devez vous familiariser avec certaines technologies Web de base et vous assurer que vous avez installé tous les outils logiciels requis.
```
Comment (dantereve) [inline docs/tutorial/part-zero/index.md]:
```suggestion
Avant de commencer à créer votre premier site Gatsby, vous devez vous familiariser avec certaines technologies web de base et vous assurer que vous avez installé tous les outils logiciels requis.
```

File: docs/tutorial/part-zero/index.md [hunk 4]
```diff
@@ -1,194 +1,195 @@
...
 
-## Familiarize yourself with the command line
+## Familiarisez-vous avec la ligne de commande
 
-The command line is a text-based interface used to run commands on your computer. You’ll also often see it referred to as the terminal. In this tutorial, we’ll use both interchangeably. It’s a lot like using the Finder on a Mac or Explorer on Windows. Finder and Explorer are examples of graphical user interfaces (GUI). The command line is a powerful, text-based way to interact with your computer.
+La ligne de commande (CLI) est une interface textuelle utilisée pour exécuter des commandes sur votre ordinateur. Vous le verrez également souvent sous le nom de terminal. Dans ce didacticiel, nous utiliserons les deux de manière interchangeable. C'est un peu comme utiliser le Finder sur un Mac ou l'Explorateur sur Windows. Finder et Explorer sont des exemples d'interfaces utilisateur graphiques (GUI). La ligne de commande est un moyen puissant et textuel d'interagir avec votre ordinateur.
```
Comment (dantereve) [inline docs/tutorial/part-zero/index.md]:
```suggestion
La ligne de commande est une interface textuelle utilisée pour exécuter des commandes sur votre ordinateur. Vous le verrez également souvent sous le nom de terminal. Dans ce didacticiel, nous utiliserons les deux de manière interchangeable. C'est un peu comme utiliser le Finder sur un Mac ou l'Explorateur sur Windows. Finder et Explorer sont des exemples d'interfaces utilisateur graphiques (GUI). La ligne de commande est un moyen puissant et textuel d'interagir avec votre ordinateur.
```

File: docs/tutorial/part-zero/index.md [hunk 5]
```diff
@@ -1,194 +1,195 @@
...
 
-Take a moment to locate and open up the command line interface (CLI) for your computer. Depending on which operating system you are using, see [**instructions for Mac**](http://www.macworld.co.uk/feature/mac-software/how-use-terminal-on-mac-3608274/), [**instructions for Windows**](https://www.quora.com/How-do-I-open-terminal-in-windows) or [**instructions for Linux**](https://www.howtogeek.com/140679/beginner-geek-how-to-start-using-the-linux-terminal/).
 
-## Install Homebrew for Node.js
+Prenez un temps pour trouver et ouvrir l'interface de ligne de commande (CLI) de votre ordinateur. Selon le système d'exploitation que vous utilisez, voir [**instructions pour Mac**](http://www.macworld.co.uk/feature/mac-software/how-use-terminal-on-mac-3608274/), [**instructions pour Windows**](https://www.quora.com/How-do-I-open-terminal-in-windows) ou [**instructions pour Linux**](https://www.howtogeek.com/140679/beginner-geek-how-to-start-using-the-linux-terminal/).
```
Comment (dantereve) [inline docs/tutorial/part-zero/index.md]:
```suggestion
Prenez le temps pour trouver et ouvrir l'interface de ligne de commande (CLI) de votre ordinateur. Selon le système d'exploitation que vous utilisez, voir [**instructions pour Mac**](http://www.macworld.co.uk/feature/mac-software/how-use-terminal-on-mac-3608274/), [**instructions pour Windows**](https://www.quora.com/How-do-I-open-terminal-in-windows) ou [**instructions pour Linux**](https://www.howtogeek.com/140679/beginner-geek-how-to-start-using-the-linux-terminal/).
```

File: docs/tutorial/part-zero/index.md [hunk 6]
```diff
@@ -1,194 +1,195 @@
...
 
-To install Gatsby and Node.js, it is recommended to use [Homebrew](https://brew.sh/). A little set-up in the beginning can save you from some headaches later on!
+## Installez Homebrew pour Node.js
 
-How to install or verify Homebrew on your computer:
+Pour installer Gatsby et Node.js, il est recommandé d'utiliser [Homebrew](https://brew.sh/). Une petite configuration au début peut vous éviter quelques maux de tête plus tard!
```
Comment (dantereve) [inline docs/tutorial/part-zero/index.md]:
```suggestion
Pour installer Gatsby et Node.js, il est recommandé d'utiliser [Homebrew](https://brew.sh/). Une petite configuration au début peut vous éviter quelques maux de tête plus tard !
```

File: docs/tutorial/part-zero/index.md [hunk 7]
```diff
@@ -1,194 +1,195 @@
...
-- Installed and learned about Node.js and the npm CLI tool, the version control system Git, and the Gatsby CLI tool
-- Generated a new Gatsby site using the Gatsby CLI tool
-- Ran the Gatsby development server and visited your site locally
-- Downloaded a code editor
-- Installed a code formatter called Prettier
+- Connaître la ligne de commande et comment l'utiliser
+- Installation et apprentissage de Node.js et de l'outil CLI npm, du système de contrôle de version Git et de l'outil CLI Gatsby
+- Généré un nouveau site Gatsby à l'aide de l'outil CLI Gatsby
+- Exécutez le serveur de développement Gatsby et visité votre site localement
+- Téléchargé un éditeur de code
```
Comment (dantereve) [inline docs/tutorial/part-zero/index.md]:
```suggestion
- Téléchargez un éditeur de code
```

**Review by dantereve — CHANGES_REQUESTED** [review]
Great work! I've found some minor typos to fix before merging. You can accept them directly from this PR. Thanks 👍.

**Comment by alexislepresle** [comment]
> Great work! I've found some minor typos to fix before merging. You can accept them directly from this PR. Thanks 👍.

Thanks Mathias 👍

**Review by dantereve — APPROVED** [review]

**Pull request merged**